manager/integration/tests/conftest.py

```diff
@@ -173,3 +177,38 @@ def pytest_collection_modifyitems(config, items):
         for item in items:
             if "upgrade" in item.keywords:
                 item.add_marker(skip_upgrade)
+
+
+def pytest_exception_interact(call, report):
```

Comment:
From the [doc](https://docs.pytest.org/en/6.2.x/reference.html#pytest.hookspec.pytest_exception_interact), please only do this hook for test report case instead of collection repot.

Reply:
Modified, please see https://github.com/longhorn/longhorn-tests/pull/879#issuecomment-1040301027